### DISCUSSION TITLE
Live Streaming questions

### DISCUSSION THREAD

**Discussion by elmau** [mozilla/hubs#5024] [Hubs Cloud Questions] [answered]
Is it possible that using a hired live streaming platform, I can send the same livestream to different rooms in a hubs-cloud server and all the people connected to them see the video? I'm thinking of possible problems that could arise from multiple people connecting to a single url and requesting the same stream in different rooms. 

Any ideas?

**Comment by pattersonbl2** [comment, marked as answer]
We currently don't have a feature to do this.  Only possible workaround is using the screen share feature to share the live stream from your local machine.  You would have to do this in each room.
